lib/epics/client.rb

```diff
@@ -1,21 +1,27 @@
 class Epics::Client
   extend Forwardable
 
-  attr_accessor :passphrase, :url, :host_id, :user_id, :partner_id, :keys, :keys_content, :locale, :product_name
-  attr_writer :iban, :bic, :name
+  attr_accessor :passphrase, :url, :host_id, :user_id, :partner_id, :keys, :keys_content, :locale, :product_name,
+                :x_509_certificate_a_content, :x_509_certificate_x_content, :x_509_certificate_e_content, :debug_mode
```

Comment:
Having individual long method names for each individual certificate does not seem very elegant.

Having the 3 keys combined under 1 method allows for keeping the immediate surface area of the client class way smaller.